docs/installing-cpython.md

```diff
@@ -259,7 +256,7 @@ build CPython using [pyenv](https://github.com/pyenv/pyenv). In order to
 do that, you can use the following:
 
 ```bash
-pyenv install --debug --keep 3.13.1
+pyenv install --debug --keep 3.14.0
```

Comment:
I think it should be just `3.14`, there are many free-threading changes coming in 3.14.1 so minor version would have to changed again.

Reply:
```suggestion
pyenv install --debug --keep 3.14
```